galaxy/values.yaml

```diff
@@ -66,10 +66,10 @@ affinity: {}
 
 postgresql:
   enabled: true
-  username: galaxydbuser
-  password: 42
+  galaxyDatabaseUser: galaxydbuser
+  galaxyDatabasePassword: 42
```

Comment:
@almahmoud I forgot one thing! We should generate a random alphanum if no database password is specified, and leave this blank by default. I think the postgres chart would be a good point of reference.